packages/govuk-frontend-review/.browserslistrc

```diff
@@ -0,0 +1,8 @@
+> 0.1%
+last 2 Chrome versions
+last 2 Firefox versions
+last 2 Edge versions
+last 2 Samsung versions
+Safari >= 9
+iOS >= 9
+ie 11
```

Comment:
We’d forgotten to add a Browserslist config to the review app during:

* https://github.com/alphagov/govuk-frontend/pull/3620

Supported browsers can be confirmed with:

```shell
npm exec --workspace govuk-frontend-review -- autoprefixer --info
```